definitions/firefox_desktop.toml

```diff
@@ -1233,6 +1233,8 @@ description = "Search Clients Engines Sources Daily"
 
 [data_sources.clients_daily]
 from_expression = "mozdata.telemetry.clients_daily"
+submission_date_column = "submission_date"
+client_id_column = "client_id"
```

Comment:
These are both the default values so I don't think they're strictly necessary, but that said I have no problem being explicit about it.